tools/java/org/unicode/cldr/api/CldrData.java

```diff
@@ -188,5 +188,14 @@ default void visitPrefixStart(CldrPath prefix, Context context) {}
          *               sub-hierarchy visitation.
          */
         default void visitPrefixEnd(CldrPath prefix) {}
+
+        /**
+         * Callback method invoked for each value encountered by this visitor. This is equivalent
+         * to the {@link ValueVisitor#visit(CldrValue)} method but is not normally needed for
+         * prefix visitors (which are expected to delegate to a separate ValueVisitor). How this
```

Comment:
The last sentence is incomplete as written. I suspect you want to remove the initial "How" and just start with "This method is useful for...".

Reply:
Yeah, not sure how that snuck in. Thanks.